tariff-management-service/app/src/main/java/com/microservices/tariffmanagement/controller/TariffController.java

```diff
@@ -0,0 +1,22 @@
+package com.microservices.tariffmanagement.controller;
+
+import com.microservices.tariffmanagement.dto.TariffDto;
+import com.microservices.tariffmanagement.service.TariffService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("tariffs")
+public class TariffController {
+    private TariffService tariffService;
+
+    public TariffController(TariffService tariffService) {
```

Comment:
@Autowired

Reply:
There is only one constructor, it is redundant. Or you want to make it super clear?

Reply:
How is it related with constructors num? Yes I want it to be explicitly specified

Reply:
If the bean have only one constructor, it can be omitted

Reply:
Anyway let's do it